src/Tempest/View/src/Elements/ViewComponentElement.php

```diff
@@ -104,7 +104,9 @@ public function compile(): string
                     $slot = $this->getSlot($name);
 
                     if ($slot === null) {
-                        return $matches[0];
+                        // A slot doesn't have any content, so we'll comment it out.
+                        // This is to prevent DOM parsing errors (slots in <head> tags is one example, see #937)
+                        return '<!--' . $matches[0] . '-->';
```

Comment:
Maybe comment it only in development, and remove it completely in production? Vue does that

Reply:
Agree: https://github.com/tempestphp/tempest-framework/issues/949